runtime/Makefile

```diff
@@ -42,24 +31,19 @@ test: BUILD_TYPE?=RelWithDebInfo
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
 	@echo "  all                to build Catalyst Runtime"
-	@echo "  runtime            to build Catalyst Runtime with PennyLane-Lightning"
 	@echo "  coverage           to generate a coverage report using lcov"
 	@echo "  clean              to delete all temporary, cache, and build files"
 	@echo "  test               to run the Catalyst runtime test suite"
 	@echo "  format [check=1]   to apply C++ formatter; use with 'check=1' to check instead of modify (requires clang-format)"
 	@echo "  format [version=?] to apply C++ formatter; use with 'version={version}' to run clang-format-{version} instead of clang-format"
 	@echo "  check-tidy         to build Catalyst Runtime with RUNTIME_CLANG_TIDY=ON (requires clang-tidy)"
 
-.PHONY: all
-all: runtime
-
 .PHONY: configure
 configure:
 	@echo "Configure Catalyst Runtime"
 
 	# TODO: Removing CMakeCache.txt before each cmake configuration will ensure that Catalyst Runtime
-	# is always built with proper cmake values. There is currently an issue with caching the
-	# PL_BACKEND variable by PennyLane-Lightning which can be fixed here:
+	# is always built with proper cmake values.
 	rm -f $(RT_BUILD_DIR)/CMakeCache.txt
```

Comment:
I think this was a lightning-specific issue, maybe we can remove it now? @maliasadi

Reply:
Removed

Reply:
Just want confirmation from Ali here

Reply:
Hi folks, this can be removed now! It was required in the past to build multiple lightning simulators in the runtime using CMake FetchContent. I would use ExternalProject if I were to rewrite this support, but it's no longer necessary since both LQ and LK are now downstreamed to Lightning :)